OracleDatabase/SingleInstance/README.md

```diff
@@ -78,7 +78,7 @@ After setting these environment variables, the container image can be built usin
 To run your Oracle Database image use the `docker run` command as follows:
 
     docker run --name <container name> \
-    -p <host port>:1521 -p <host port>:5500 \
+    -p <host port>:1521 -p <host port>:5500 -p <host port>: 1522\
```

Comment:
Is the space after the colon deliberate?

Reply:
It was not deliberate. Fixed.